### PR TITLE
Williams System 11B: initial maps

### PR DESCRIPTION
**Pull request by francisdb**
Went through through this list: https://www.ipdb.org/search.pl?searchtype=advanced&mpu=8

### CODE REVIEW

File: gmine_l2.nv.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
+{
+  "_notes": [
+    "Compiled by Francis De Brabandere.",
+    "Space Station (Williams 1987)",
+    "https://www.ipdb.org/machine.cgi?id=2261",
+    "Williams System 11B"
+  ],
```
Comment (tomlogic) [inline gmine_l2.nv.json:7]: Needs update to correct game name and ROM name.

File: gmine_l2.nv.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  "_version": 0.1,
+  "game_state": {
+    "credits": {
+      "label": "Credits",
+      "start": "0x44",
+      "length": 1,
+      "encoding": "bcd"
+    },
+    "default_high_score": {
+      "_note": "Should probably be a adjustment, this is the default high score when pressing the highscore reset button (5 in vpinball). Default is 200",
```
Comment (tomlogic) [inline gmine_l2.nv.json:24]: You probably don't want to try figuring out all adjustments, but you could start an adjustments section and list this as 12.  Check the High Speed JSON for the Adjustments section.

**Review by tomlogic — COMMENTED** [review]
You might want to look at `tsptr_l3.nv.json` to see how it compares to other 11B games, and whether the standard audits are consistent across games.
When I've played with audits, I've used a hex editor to write different values to each possible audit field (they're usually grouped) and then see how they show up in the service menu.  My Python code includes functions to correct checksums, assuming they're properly listed in the JSON file.

File: gmine_l2.nv.json (hunk 2 above)
Comment (francisdb) [reply gmine_l2.nv.json:24]: for now removing this as I want to first get all high scores done

**Comment by francisdb** [comment]
For now leaving adjustments out

**Pull request merged**